rest-server/src/config/job.js

```diff
@@ -54,6 +54,31 @@ const jobConfigSchema = Joi.object().keys({
         gpuNumber: Joi.number()
           .integer()
           .default(0),
+        portList: Joi.array()
+          .items(Joi.object().keys({
+            label: Joi.string()
+              .regex(/^[A-Za-z0-9\-._~]+$/)
+              .required(),
+            beginAt: Joi.number()
+              .integer()
+              .default(0),
+            portNumber: Joi.number()
+              .integer()
+             .default(1),
+          }))
+          .optional()
+          .default([
+            {
+              label: 'http',
+              beginAt: 0,
+              portNumber: 1,
+            },
+            {
+              label: 'ssh',
+              beginAt: 0,
+              portNumber: 1,
+            },
+          ]),
         command: Joi.string()
```

Comment:
If user don't specific portList, it will use the default http,ssh settings. 

if user only set one port list, for example only set http, but default the ssh port will ingored ?

Reply:
yes